src/main/java/com/zebrunner/carina/webdriver/DriverHelper.java

```diff
@@ -496,6 +506,192 @@ public void addCookie(Cookie cookie) {
         });
     }
 
+    /**
+     * add a cookie object into the driver
+     *
+     * @param name name of the cookie
+     * @param value the value of the cookie
+     * @param domain the domain of the cookie
+     * @param path the path of the cookie
+     * @param expiry when the cookie expires
+     */
+    public void addCookie(String name, String value, String domain, String path, Date expiry) {
+        addCookie(new Cookie(name, value, domain, path, expiry));
+    }
+
+    /**
+     * add a cookie object into the driver
+     *
+     * @param name name of the cookie
+     * @param value the value of the cookie
+     * @param domain the domain of the cookie
+     * @param path the path of the cookie
+     * @param expiry when the cookie expires
+     * @param isSecure if the cookie is secure or not
+     * @param isHttpOnly if it is on http or https
+     */
+    public void addCookie(String name, String value, String domain, String path, Date expiry, boolean isSecure, boolean isHttpOnly) {
+        addCookie(new Cookie(name, value, domain, path, expiry, isSecure, isHttpOnly));
+    }
+
+    /**
+     * gets a specific cookie
+     *
+     * @param name the cookie being targeted
+     * @return cookie with the given name
+     * @see WebDriver.Options#getCookieNamed(String)
+     */
+    public Cookie getCookieNamed(String name) {
+        return getDriver().manage().getCookieNamed(name);
+    }
+
+    /**
+     * get the value of a cookie that was a URL-encoded Map.
+     *
+     * @param name the cookie being targeted
+     * @return decoded cookie value
+     */
+    @SuppressWarnings("unchecked")
+    public Map<String, String> getDecodedValueOfCookieNamed(String name) {
```

Comment:
What means URL-encoded Map?
Provide additional info

Reply:
Looking through our use cases and the needs of Carina I think I'm going to just remove this method.